products.d/tumbleweed.yaml

```diff
@@ -47,6 +47,16 @@ software:
   mandatory_patterns:
     - enhanced_base # only pattern that is shared among all roles on TW
   optional_patterns: null # no optional pattern shared
+  user_patterns:
+    - basic_desktop
+    - xfce
+    - kde
+    - gnome
+    - yast2_basis
+    - yast2_desktop
+    - yast2_server
```

Comment:
I am not sure if it makes sense to have all three. Maybe just desktop and server yast modules will be enough?

Reply:
I'm not sure. If you want to have YaST just for the package management then the `yast2_basis` is enough. On the other hand the "YaST Desktop" adds just few megabytes more, not a big deal...